aggregation-worker/pom.xml

```diff
@@ -86,24 +88,67 @@
         </repository>
     </repositories>
 
+    <distributionManagement>
+        <repository>
+            <id>awsdocker</id>
+            <url>https://${aws.docker.repository.uri}</url>
+        </repository>
+    </distributionManagement>
+
     <build>
         <plugins>
+            <!-- Set up JAR manifest -->
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
+                            <classpathPrefix>lib/</classpathPrefix>
+                            <mainClass>au.org.emii.aggregationworker.Application</mainClass>
+                        </manifest>
+                    </archive>
+                </configuration>
+            </plugin>
+            <!-- Copy Maven dependencies into target/lib/ -->
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <overWriteReleases>false</overWriteReleases>
+                            <includeScope>runtime</includeScope>
+                            <outputDirectory>${project.build.directory}/lib</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
-                <version>2.3</version>
+                <groupId>com.spotify</groupId>
+                <artifactId>dockerfile-maven-plugin</artifactId>
+                <version>1.3.5</version>
                 <configuration>
-                    <createDependencyReducedPom>false</createDependencyReducedPom>
+                    <repository>${aws.docker.repository.uri}</repository>
+                    <googleContainerRegistryEnabled>false</googleContainerRegistryEnabled>
+                    <verbose>true</verbose>
+                    <tag>latest</tag>
                 </configuration>
                 <executions>
                     <execution>
+                        <id>build-tag-push-image</id>
                         <phase>package</phase>
                         <goals>
-                            <goal>shade</goal>
+                            <goal>build</goal>
```

Comment:
Not sure these are quite right - its trying to push the image whenever I try and build it - e.g. using package - shouldn't pushing the image just be a deploy thing?  otherwise we'll be pushing everytime we build the project when devloping.